HW_6/task_2.py

```diff
@@ -0,0 +1,21 @@
+"""Добавьте в пакет, созданный на семинаре шахматный модуль. Внутри него напишите код,
+решающий задачу о 8 ферзях. Известно, что на доске 8×8 можно расставить 8 ферзей так,
+чтобы они не били друг друга. Вам дана расстановка 8 ферзей на доске, определите,
+есть ли среди них пара бьющих друг друга. Программа получает на вход восемь пар чисел,
+каждое число от 1 до 8 - координаты 8 ферзей. Если ферзи не бьют друг друга верните истину,
+ а если бьют - ложь."""
+queens = [(1, 7), (2, 4), (3, 2), (4, 8), (5, 6), (6, 1), (7, 3), (8, 5)]
+
+IS_SAFE = True
+
+for i in range(8):
+    for j in range(i + 1, 8):
+        if queens[i][0] == queens[j][0] or queens[i][1] == queens[j][1] \
+                or abs(queens[i][0] - queens[j][0]) == abs(queens[i][1] - queens[j][1]):
+            IS_SAFE = False
+            break
+
+if IS_SAFE:
+    print("True")
+else:
+    print("False")
```

Comment:
Можно использовать связку `break` и `else` от цикла.
```suggestion
for i in range(8):
    for j in range(i + 1, 8):
        if queens[i][0] == queens[j][0] or queens[i][1] == queens[j][1] \
                or abs(queens[i][0] - queens[j][0]) == abs(queens[i][1] - queens[j][1]):
            print("False")
            break

else:
    print("True")
```